jwala-services/src/main/java/com/cerner/jwala/service/group/impl/GroupJvmControlServiceImpl.java

```diff
@@ -75,4 +75,16 @@ public CommandOutput call() throws Exception {
             }
         }
     }
+
+    public boolean checkSameState(final JvmControlOperation jvmControlOperation, final JvmState jvmState) {
+        return (jvmControlOperation.START.toString().equals(jvmControlOperation.name()) && jvmState.isStartedState()) ||
+                (jvmControlOperation.STOP.toString().equals(jvmControlOperation.name()) && !jvmState.isStartedState()) ? true : false;
```

Comment:
Remove the "? true : false" at the end of this conditional - it's already implied